src/Controls/tests/UITests/Tests/Issues/Issue17694.cs

```diff
@@ -0,0 +1,33 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue17694 : _IssuesUITest
+	{
+		public Issue17694(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "Circle view not rotating from center";
+
+		[Test]
+		public void Issue17694Test()
+		{
+			this.IgnoreIfPlatforms(new[]
+			{
+				TestDevice.iOS,
+				TestDevice.Mac,
+				TestDevice.Android
+			});
+
+			App.WaitForElement("Spin");
+
+			// 1. Click button
+			App.Click("Spin");
+
+			VerifyScreenshot();
```

Comment:
The added test is failing. I think we need to wait a little bit before verify the screenshot to avoid possible problems with the animation still running.
![image](https://github.com/dotnet/maui/assets/6755973/9a035bb0-bd9a-46e8-8d1c-829bf30a9096)
`Snapshot different than baseline: Issue17694Test.png (4.40% difference)`

Reply:
Ah, and the mica background I think is causing issues